packages/cypress/src/bin/archive-storybook.ts

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
```

Comment:
We could probably move these script files back into shared and include them in the symlinked storybook config dir, but can decide on that later.